Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4440,7 +4453,22 @@ AXTextChange AXObjectCache::textChangeForEditType(AXTextEditType type)
     return AXTextInserted;
 }
 #endif
-    
+
+AccessibilityObject* AXObjectCache::textCompositionObjectForNode(Node& node)
+{
+    // Find the ax object that sends text change notifications for the composition
```

Comment:
This comment seems redundant thanks to your descriptive function name — can probably remove it.

Reply:
removed